core/shared/src/main/scala-2/zio/prelude/Newtype.scala

```diff
@@ -306,4 +308,23 @@ object Newtype {
     val _ = newtype
     value.asInstanceOf[F[T#Type]]
   }
+
+  private trait ClassTagWrapper[-A] {
+    def classTag: ClassTag[_]
+  }
+
+  private object ClassTagWrapper {
+    def apply[A](implicit ev: ClassTagWrapper[A]): ClassTagWrapper[A] = ev
+
+    implicit def classTagWrapperForNewtype[A](implicit underlying: ClassTag[A]): ClassTagWrapper[Newtype[A]] =
+      new ClassTagWrapper[Newtype[A]] {
+        def classTag: ClassTag[_] = underlying
+      }
+  }
+
+  implicit def classTag[N <: Newtype[_]: ClassTagWrapper]: ClassTag[N#Type] = {
```

Comment:
I'm not quite sure why we need the implicit def both here and in the trait, but 2.12 and 2.13 fail unless both are present.
In practice they will only evaluate this one, so it seems to just affect implicit search in some way.